static/js/directives/chat.js

```diff
@@ -383,7 +383,7 @@ define(['jquery', 'underscore', 'text!partials/chat.html', 'text!partials/chatro
 								// Make sure we are not in group chat or the message is from ourselves
 								// before we beep and shout.
 								if (!subscope.isgroupchat && from !== sessionid) {
-									playSound.play("message1");
+									playSound.play("announce");
```

Comment:
When checking for sounds on my side it makes it more readable if the key for the sound is used. When all instance use the key then I just have to check for the key and not for the value as well!